src/network/Server.php

```diff
@@ -44,4 +44,7 @@ public function nextConnectionNDAsync(): Awaitable<TNDSock>;
 
   /** Return the local (listening) address for the server */
   public function getLocalAddress(): this::TAddress;
+
+  /** Stop listening; open connection are not closed */
+  public function stopListening(): void;
```

Comment:
just ... `stop(): void` ?

Reply:
I was worried that might imply that it stops active connections, which is not the case.

Reply:
makes sense :+1: